_targets.R

```diff
@@ -7,36 +7,48 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
+    site_01427207,
+    download_nwis_data(site_num = "01427207"),
+  ),
+  tar_target(
+    site_01432160,
+    download_nwis_data(site_num = "01432160"),
+  ),
+  tar_target(
+    site_01435000,
+    download_nwis_data(site_num = "01435000"),
+  ),
+  tar_target(
+    site_01436690,
+    download_nwis_data(site_num = "01436690"),
+  ),
+  tar_target(
+    site_01466500,
+    download_nwis_data(site_num = "01466500"),
+  ),
   tar_target(
     site_data,
-    download_nwis_data(),
+    bind_rows(site_01427207, site_01432160, site_01435000, site_01436690, site_01466500),
   ),
   tar_target(
     site_info_csv,
-    nwis_site_info(fileout = "1_fetch/out/site_info.csv", site_data),
+    nwis_site_info(fileout = "1_fetch/out/site_info.csv", site_01427207, site_01432160, site_01435000, site_01436690, site_01466500),
```

Comment:
I think you could leave this using `site_data` since that uses `bind_rows()` for all of these and you are redoing that step as the first part of this `nwis_site_info()` function.

Reply:
Good point